packages/blockchain-wallet-v4/src/types/KVStoreEntry.js

```diff
@@ -61,19 +61,19 @@ export const fromKeys = (entryECKey, encKeyBuffer, typeId) => {
   })
```

Comment:
I don't remember if kvstore works with testnet addresses. Have you tried that @plondon ?

Reply:
I actually need to check this more because if I set a kvstore value (e.g for shapeshift state) then login breaks.

Note to self:
```
at fromObject (index.js:311)
    at from (index.js:137)
    at Function.push.../../node_modules/buffer/index.js.Buffer.from (index.js:149)
    at magicHash (index.js:37)
    at Object.verify (index.js:57)
    at KVStoreEntry.js:136
    at _curryN.js:34
    at _arity.js:22
    at KVStoreEntry.js:160
    at _curryN.js:34
```

Reply:
kvstore works now